plux/build/setuptools.py

```diff
@@ -375,6 +409,27 @@ def _to_filename(name):
     return name.replace("-", "_")
 
 
+def _path_to_module(path):
+    """
+    Convert a path to a Python module to its module representation
+    Example: plux/core/test -> plux.core.test
+    """
+    return path.strip("/").replace("/", ".")
```

Comment:
perhaps we can use pathlib to remove some assumptions about path structure

with `from pathlib import Path`

```suggestion
    return '.'.join(Path(path).with_suffix('').parts)
```